.travis.yml

```diff
@@ -69,9 +69,9 @@ matrix:
       env: PYTHON="3" SCALA_VER="2.11" SPARK_VER="2.2.0" HADOOP_VER="2.6" PROFILE="-Pspark-2.2 -Pweb-ci -Pscalding -Phelium-dev -Pexamples -Pscala-2.11" BUILD_FLAG="package -Pbuild-distr -DskipRat" TEST_FLAG="verify -Pusing-packaged-distr -DskipRat" MODULES="-pl ${INTERPRETERS}" TEST_PROJECTS="-Dtests.to.exclude=**/ZeppelinSparkClusterTest.java,**/org.apache.zeppelin.spark.*,**/HeliumApplicationFactoryTest.java -DfailIfNoTests=false"
 
     # Test selenium with spark module for 1.6.3
-    - jdk: "oraclejdk7"
+    - jdk: "oraclejdk8"
```

Comment:
my concern is spark 1.6.3 doesn't officially support jdk 8
https://spark.apache.org/docs/1.6.3/#downloading

Reply:
Should I change both Scala and Spark to `SCALA_VER="2.11" SPARK_VER="2.2.0"`? As selenium-3.x requires JDK 8. However, I see travis CI (https://travis-ci.org/prabhjyotsingh/zeppelin/jobs/326501440/config) is passing.

Reply:
You can keep JDK7 here. But only use JDK8 when running selenium test.